spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'byebug'
```

Comment:
It's already been declared here.

https://github.com/Fablic/fablicop/blob/d103863d2499c0ffbbd3aea8ae658c9780e8d51d/spec/spec_helper.rb#L6